src/comments/models.py

```diff
@@ -60,10 +60,12 @@ def create_comment(self, user=None, text=None, path=None, target=None, parent=No
         if parent is not None:
             comment.parent = parent
 
+        comment.full_clean()
         comment.save(using=self._db)
```

Comment:
Why are we saving twice?  Can this one be removed?

Reply:
nope. the first save is to initialize the object in the db (in this case its to get id.) Then the next line of code is to calculate the path which uses id.

ill remove the first `full_clean()` since its redundant